backend/de.metas.business/src/main/java/de/metas/requisition/interceptor/M_RequisitionLine.java

```diff
@@ -48,4 +63,29 @@ public void updateIsVendor(@NonNull final I_M_RequisitionLine requisitionLine)
 			requisitionLine.setIsVendor(bpartner.isVendor());
 		}
 	}
+
+	@ModelChange(timings = {ModelValidator.TYPE_AFTER_CHANGE }, ifColumnsChanged = { I_M_RequisitionLine.COLUMNNAME_LineNetAmt })
+	public void updateRequisitionTotalLines(@NonNull final I_M_RequisitionLine line)
+	{
+		calculateTotalLines(line);
```

Comment:
hmmm, this won't work nice in case multiple lines are updated in the same transaction 
or in case a new line is created
or a line is deleted

Reply:
pls consider using org.adempiere.ad.trx.api.ITrxManager#accumulateAndProcessAfterCommit

Reply:
e.g.
![image](https://github.com/metasfresh/metasfresh/assets/1244701/47b0db03-f182-4b5c-a7b9-38d430254594)